at-quannguyen/W3/exercise_2.rb

```diff
@@ -0,0 +1,5 @@
+def sum_is_integer(nums)
+  nums.all?(Integer) ? nums.inject(0, :+) : false
+end
```

Comment:
Define method bên trong class Array
Sau đó chạy code như thế này để in ra kết quả `[1,2,3,4].sum # 10`